### PR TITLE
Add higby.io

### PR DESCRIPTION
**Pull request by higby**

### CODE REVIEW

File: 2024.html [hunk 1]
```diff
@@ -95,6 +95,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://blakewatson.com/">Blake Watson</a></li>
             <li><a href="https://kayserifserif.place/">Katherine Yang</a></li>
             <li><a href="https://cssence.com/">Matthias Zöchling</a></li>
+            <li><a href="https://www.higby.io/">Branden Higby</a></li>
```
Comment (j9t) [inline 2024.html:98]: Please move up (the list is sorted by last name).

**Review by j9t — CHANGES_REQUESTED** [review]

File: 2024.html (hunk 1 above)
Comment (arkhi) [inline 2024.html:98]: Hello!

You can follow the instructions about sorting in the [README of this repository](https://github.com/css-naked-day/css-naked-day.github.io) (or the [comment above the list](https://github.com/css-naked-day/css-naked-day.github.io/pull/153/files#diff-24ac768f3b5003cd704e67986b8c414fa8320442ba2821cb55a2e9242d54c3e4L53-L59) :) ).

Thank you!

Comment (higby) [reply 2024.html]: My apologies, I mistakenly appended it. It should be fixed now.

**Review by j9t — APPROVED** [review]

**Comment by j9t** [comment]
Welcome aboard!

**Pull request merged**